dockers/release/Dockerfile

```diff
@@ -36,7 +36,7 @@ RUN \
         mv pytorch-lightning-*/ pytorch-lightning ; \
         rm *.zip ; \
     fi && \
-    pip install ./pytorch-lightning["extra"] --no-cache-dir && \
+    pip install ./pytorch-lightning["extra","loggers","strategies"] --no-cache-dir --upgrade-strategy only-if-needed && \
```

Comment:
Isn't "only-if-needed" the default upgrade strategy anyways?

Reply:
good question...

Reply:
```suggestion
    pip install ./pytorch-lightning["extra","loggers","strategies"] --no-cache-dir && \
```